packages/vertica-nodejs/lib/connection.js

```diff
@@ -46,9 +47,9 @@ class Connection extends EventEmitter {
     this.tls_config = config.tls_config
 
     if (this.tls_config === undefined) {
-      this.tls_mode = config.tls_mode || 'disable'
-      //this.tls_client_key = config.tls_client_key
-      //this.tls_client_cert = config.tls_client_cert
+      this.tls_mode = config.tls_mode || 'prefer'
+      this.tls_client_key = config.tls_client_key
```

Comment:
I don't think this needs to be uncommented, does it? I added these comments in as breadcrumbs for when we support mutual TLS, but the client key and client cert won't be necessary for the changes we have made

Reply:
I commented this out because it was previously implemented. However, since this is just a placeholder, I thought it best to leave it as is instead of removing it completely. It can be used during mutual TLS implementation.